ocfweb/docs/docs/services/web/flask.md

```diff
@@ -23,13 +23,21 @@ OCF servers requires only just a little extra configuration.
 
 * run.fcgi:
 
-      #!/usr/bin/env python
+      #!venv/bin/python
       from flup.server.fcgi import WSGIServer
       from flaskyapplication import app
 
       if __name__ == '__main__':
           WSGIServer(app).run()
 
+1.     Make a virtualenv.  This will allow you to *pip install* any python packages
+that you need that the OCF does not already have:
```

Comment:
maybe add a line like "The benefit is that you can choose the exact versions of dependencies you want, without being tied to the system versions that all OCF accounts use."